packages/rx_bloc_cli/example/docs/continuous_delivery.md

```diff
@@ -23,7 +23,24 @@ export MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD=<PASSWORD>
 
 Before committing any changes, make sure you encrypt all the files using the `encode.sh` script.
 
-### Fastfile amendments
+### Project & Fastfile amendments
+
+For the iOS project, make sure you set project signing to manual and apply expected provisioning profile for each build configuration.
+To do that, you need to:
+- Open project in XCode
+- Inside the targets section, select your `Runner` target
+- Open the `Signing & Capabilities` tab
+- Uncheck the `Automatically manage signing` checkmark in each of the build configurations
+- Select a fitting provisioning profile for each of the build configurations
+- Save the changes in XCode
+
+Inside the `{project_root}/bin/` directory, you'll find several scripts generated by the rx_bloc_cli package.
+Among others, make sure to find the `build_runner_build.sh` which is used during CI/CD pipeline. 
+Change the execution privileges of that file using the following command:
+
+```
+sudo chmod 755 path/to/file
```

Comment:
Why we need to use sudo ? Can't we execute this automatically from the rx_bloc_cli crate ? chmod +x

Reply:
Should be addressed through code generation now...